src/renderer/pages/torrent-list-page.js

```diff
@@ -387,3 +376,8 @@ function getErrorMessage (torrentSummary) {
   }
   return 'Error'
 }
+
+// Returns '1.9m', '205k', etc
+function prettySpeed (n) {
+  return prettyBytes(n).replace('B', '').toLowerCase()
```

Comment:
Not a fan of the lowercase because we use MB/GB in the torrent size, earlier on the status line